.github/workflows/release.yaml

```diff
@@ -24,34 +27,33 @@ jobs:
           path: node_modules
           key: ${{ runner.OS }}-build-${{ hashFiles('package.json') }}
 
-      - name: Install 🔧 # Installs dependencies
+      - name: Install 🔧
         run: yarn install --immutable
 
       - name: Build
         run: yarn build
 
   publish:
     if: ${{ !contains(github.event.head_commit.message, '[skip-ci]') &&
-      (
-        (github.ref == 'refs/heads/master' && contains(github.event.head_commit.message, '[release]')) ||
-        (startsWith(github.ref, 'refs/heads/release-') &&
-          (startsWith(github.event.head_commit.message, 'fix:') ||
-          startsWith(github.event.head_commit.message, 'fix(') ||
-          startsWith(github.event.head_commit.message, 'feat:') ||
-          startsWith(github.event.head_commit.message, 'feat(') ||
-          contains(github.event.head_commit.message, '[release]')
+          (
+            (github.ref == 'refs/heads/master' && contains(github.event.head_commit.message, '[release]')) ||
+            (startsWith(github.ref, 'refs/heads/release-') &&
+              (
+                startsWith(github.event.head_commit.message, 'fix:') ||
+                startsWith(github.event.head_commit.message, 'feat:') ||
+                contains(github.event.head_commit.message, '[release]')
```

Comment:
some extra lines touched here. e.g. `startsWith(github.event.head_commit.message, 'feat('` should stay